builder/builder.go

```diff
@@ -155,6 +159,47 @@ func (b *Builder) BuildTarget(ctx context.Context, target domain.Target, opt Bui
 	return mts, nil
 }
 
+func (b *Builder) startRegistryProxy(ctx context.Context, caps apicaps.CapSet) (func(), bool) {
+
+	if err := caps.Supports(pb.CapEarthlyRegistryProxy); err != nil {
+		b.opt.Console.VerbosePrintf("Registry proxying not supported by BuildKit: %s", err)
```

Comment:
nit: you might want to use VerboseWarnf
```suggestion
		b.opt.Console.VerboseWarnf("Registry proxying not supported by BuildKit: %v", err)
```

Reply:
I think I read this too quickly initially, `VerboseWarnf` is still flagged via verbose, (I read it simply as warnf). Either seems fine in my opinion.